README.md

```diff
@@ -18,7 +18,65 @@ with Bazel.
 
 ## Setup
 
-To be documented.
+### Using npm and Node.js rules
+
+We recommend you use npm to install the PostCSS build rules if you use the
+[Node.js build rules][rules_nodejs], and **strongly recommend** using npm
+if you also depend or plan to depend on PostCSS in your local `package.json`.
+(For example, if PostCSS plugin(s) will form part of your repository's build
+pipeline.)
+
+Installing the PostCSS build rules via npm allows you to rely on its native
+package management functionality. If you depend on PostCSS in your local
+`package.json`, using npm also helps avoid version skew between PostCSS in
+your repository and these build rules.
+
+[rules_nodejs]: https://bazelbuild.github.io/rules_nodejs/
+
+Add the `@bazel/postcss` npm package to your `devDependencies` in
+`package.json`.
+
+Your `WORKSPACE` should declare a `yarn_install` or `npm_install` rule named
+`npm`. It should then install the rules found in the npm packages using the
```

Comment:
We have deprecated the install_bazel_dependencies function in rules_nodejs 2.0, due in a week or so.
The reason is that load()ing from `@npm` causes eager install from npm even for builds that don't request any frontend artifacts.

Reply:
Removed